Warning: no access to tty (Bad file descriptor).
Thus no job control in this shell.
Thu Oct 20 07:57:45 GMT 2022
jlab.py:30: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  elif dashinfo is not '0':
jlab.py:33: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if dashinfo is not '0':
Fail to get yarn configuration. {"type":"error","data":"An unexpected error occurred: \"ENOENT: no such file or directory, open '/home1/.yarnrc'\"."}
{"type":"info","data":"If you think this is a bug, please open a bug report with the information provided in \"/home1/datahome/mdemol/.miniconda3/envs/m2env/lib/python3.8/site-packages/jupyterlab/yarn-error.log\"."}
{"type":"info","data":"Visit https://yarnpkg.com/en/docs/cli/config for documentation about this command."}

[I 2022-10-20 07:58:14.352 ServerApp] jupyterlab | extension was successfully linked.
[W 2022-10-20 07:58:14.364 NotebookApp] 'password' has moved from NotebookApp to ServerApp. This config will be passed to ServerApp. Be sure to update your config before our next release.
[I 2022-10-20 07:58:14.371 ServerApp] nbclassic | extension was successfully linked.
[I 2022-10-20 07:58:20.439 ServerApp] notebook_shim | extension was successfully linked.
[I 2022-10-20 07:58:21.868 ServerApp] notebook_shim | extension was successfully loaded.
[I 2022-10-20 07:58:21.869 LabApp] JupyterLab extension loaded from /home1/datahome/mdemol/.miniconda3/envs/m2env/lib/python3.8/site-packages/jupyterlab
[I 2022-10-20 07:58:21.869 LabApp] JupyterLab application directory is /home1/datahome/mdemol/.miniconda3/envs/m2env/share/jupyter/lab
[I 2022-10-20 07:58:21.873 ServerApp] jupyterlab | extension was successfully loaded.
[I 2022-10-20 07:58:22.098 ServerApp] nbclassic | extension was successfully loaded.
[I 2022-10-20 07:58:22.098 ServerApp] Serving notebooks from local directory: /home1/datahome/mdemol
[I 2022-10-20 07:58:22.098 ServerApp] Jupyter Server 1.16.0 is running at:
[I 2022-10-20 07:58:22.098 ServerApp] http://r1i7n32:8877/lab
[I 2022-10-20 07:58:22.098 ServerApp]  or http://127.0.0.1:8877/lab
[I 2022-10-20 07:58:22.098 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 2022-10-20 08:01:51.508 LabApp] Build is up to date
[W 2022-10-20 08:01:54.219 ServerApp] Notebook drifters_stats/sandbox/overview.ipynb is not trusted
[I 2022-10-20 08:01:55.596 ServerApp] Kernel started: 8566a462-1504-4dd8-8059-70ca8b08e2ed
[I 2022-10-20 08:01:55.606 ServerApp] Kernel started: ec78af4b-f81e-4bc8-83c2-0fa64cefe642
readline: readline: /etc/inputrc: line 19: term: unknown variable name
/etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 0cbcc8ab-8c34-4ec0-936e-c936fa6e31d0
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: dbbdb468-1801-4a69-8758-78aec598ea5c
[I 2022-10-20 08:03:54.920 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 08:04:25.004 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 08:04:36.667 ServerApp] Kernel started: c7732078-f7da-4834-a7b9-3048b41e0b41
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 1008672a-59f2-4f37-a312-eb7054400851
[I 2022-10-20 08:06:35.635 ServerApp] Saving file at /drifters_stats/sandbox/errors_analysis.ipynb
[W 2022-10-20 08:09:51.922 ServerApp] WebSocket ping timeout after 119999 ms.
[W 2022-10-20 08:09:51.923 ServerApp] WebSocket ping timeout after 119999 ms.
[W 2022-10-20 08:09:51.964 ServerApp] WebSocket ping timeout after 119997 ms.
[I 2022-10-20 08:09:56.926 ServerApp] Starting buffering for ec78af4b-f81e-4bc8-83c2-0fa64cefe642:fcd7354c-0487-48c9-86a8-254ea1658a92
[I 2022-10-20 08:09:56.965 ServerApp] Starting buffering for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[W 2022-10-20 08:10:07.635 ServerApp] WebSocket ping timeout after 119997 ms.
[I 2022-10-20 08:10:12.638 ServerApp] Starting buffering for c7732078-f7da-4834-a7b9-3048b41e0b41:fa553a6f-8be2-4645-936a-dc17befc243c
[I 2022-10-20 08:15:51.896 ServerApp] Restoring connection for ec78af4b-f81e-4bc8-83c2-0fa64cefe642:fcd7354c-0487-48c9-86a8-254ea1658a92
[I 2022-10-20 08:15:51.902 ServerApp] Restoring connection for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: ba26986d-3e45-4dc3-8aa8-3f3e9b18eb25
[IPKernelApp] WARNING | No such comm: e2943432-a302-4759-b4b3-997058022ec8
[I 2022-10-20 08:15:51.947 ServerApp] Restoring connection for c7732078-f7da-4834-a7b9-3048b41e0b41:fa553a6f-8be2-4645-936a-dc17befc243c
[I 2022-10-20 08:15:51.953 ServerApp] Replaying 4 buffered messages
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: ef61a75b-36e6-41cc-ba15-e133bc6791c2
[I 2022-10-20 08:17:25.447 ServerApp] Saving file at /drifters_stats/sandbox/errors_analysis.ipynb
[W 2022-10-20 08:19:51.898 ServerApp] WebSocket ping timeout after 119998 ms.
[W 2022-10-20 08:19:51.900 ServerApp] WebSocket ping timeout after 119998 ms.
[W 2022-10-20 08:19:51.902 ServerApp] WebSocket ping timeout after 119999 ms.
[W 2022-10-20 08:19:51.947 ServerApp] WebSocket ping timeout after 119998 ms.
[I 2022-10-20 08:19:56.901 ServerApp] Starting buffering for ec78af4b-f81e-4bc8-83c2-0fa64cefe642:6014b004-34aa-47c1-9f80-22bc56cccffc
[I 2022-10-20 08:19:56.903 ServerApp] Starting buffering for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[I 2022-10-20 08:19:56.948 ServerApp] Starting buffering for c7732078-f7da-4834-a7b9-3048b41e0b41:fa553a6f-8be2-4645-936a-dc17befc243c
[I 2022-10-20 08:20:06.596 ServerApp] Restoring connection for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[I 2022-10-20 08:20:06.603 ServerApp] Restoring connection for c7732078-f7da-4834-a7b9-3048b41e0b41:fa553a6f-8be2-4645-936a-dc17befc243c
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 371143dc-8ad0-4189-b974-ddda0ddd1148
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 8e46a168-07fc-4dbe-9749-f6c2ca170679
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 473e0b81-7b27-415e-a142-c9fb280bd523
[W 2022-10-20 08:25:06.600 ServerApp] WebSocket ping timeout after 119999 ms.
[W 2022-10-20 08:25:06.602 ServerApp] WebSocket ping timeout after 119999 ms.
[W 2022-10-20 08:25:06.604 ServerApp] WebSocket ping timeout after 119999 ms.
[W 2022-10-20 08:25:06.604 ServerApp] WebSocket ping timeout after 119999 ms.
[I 2022-10-20 08:25:11.602 ServerApp] Starting buffering for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[I 2022-10-20 08:25:11.605 ServerApp] Starting buffering for c7732078-f7da-4834-a7b9-3048b41e0b41:fa553a6f-8be2-4645-936a-dc17befc243c
[I 2022-10-20 08:25:11.605 ServerApp] Starting buffering for ec78af4b-f81e-4bc8-83c2-0fa64cefe642:6014b004-34aa-47c1-9f80-22bc56cccffc
[I 2022-10-20 08:35:07.462 ServerApp] Restoring connection for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[I 2022-10-20 08:35:07.465 ServerApp] Restoring connection for c7732078-f7da-4834-a7b9-3048b41e0b41:fa553a6f-8be2-4645-936a-dc17befc243c
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 21462860-4494-422d-a5c5-275a3c0b5ff0
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: b73ae100-08f0-4530-a02b-bce94729c30f
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 8f04b2df-aedf-4610-b660-99638ed68230
[I 2022-10-20 08:36:19.663 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 08:37:45.640 ServerApp] Saving file at /drifters_stats/sandbox/errors_analysis.ipynb
[I 2022-10-20 08:37:47.890 ServerApp] Starting buffering for c7732078-f7da-4834-a7b9-3048b41e0b41:fa553a6f-8be2-4645-936a-dc17befc243c
[W 2022-10-20 08:38:10.620 ServerApp] delete /drifters_stats/sandbox/dask-worker.o2408494
[W 2022-10-20 08:38:10.810 ServerApp] delete /drifters_stats/sandbox/dask-worker.o2408495
[W 2022-10-20 08:38:10.814 ServerApp] delete /drifters_stats/sandbox/dask-worker.o2408496
[W 2022-10-20 08:38:10.817 ServerApp] delete /drifters_stats/sandbox/dask-worker.o2408497
[W 2022-10-20 08:38:10.820 ServerApp] delete /drifters_stats/sandbox/dask-worker.o2408498
[I 2022-10-20 08:38:19.778 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[W 2022-10-20 08:39:31.405 ServerApp] Notebook drifters_stats/sandbox/pyinterp.ipynb is not trusted
[I 2022-10-20 08:39:31.587 ServerApp] Kernel started: 0b9d85d6-8c2f-4aff-a016-569d3a83a0c4
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: e39029bc-bbc1-494f-bbfd-487227a34734
[I 2022-10-20 08:40:19.888 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 08:41:31.473 ServerApp] Saving file at /drifters_stats/sandbox/pyinterp.ipynb
[W 2022-10-20 08:41:31.474 ServerApp] Notebook drifters_stats/sandbox/pyinterp.ipynb is not trusted
[W 2022-10-20 08:54:57.698 ServerApp] Notebook pynsitu/dev/overview.ipynb is not trusted
[I 2022-10-20 08:54:58.686 ServerApp] Kernel started: 4973cfa3-ee08-4aaa-86fd-cd75f65f51fc
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 3f323ea6-b147-470a-9bfc-94fc3481c0ee
[W 2022-10-20 08:56:09.362 ServerApp] Notebook pynsitu/dev/clean.ipynb is not trusted
[I 2022-10-20 08:56:11.447 ServerApp] Kernel started: 2b65cd5b-80b7-472e-89d6-b084ccc92d78
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: d199ec80-cd9a-4f85-8ea0-07ffce1b099f
[I 2022-10-20 08:56:58.075 ServerApp] Saving file at /pynsitu/dev/overview.ipynb
[W 2022-10-20 08:56:58.075 ServerApp] Notebook pynsitu/dev/overview.ipynb is not trusted
[I 2022-10-20 08:58:10.153 ServerApp] Saving file at /pynsitu/dev/clean.ipynb
[W 2022-10-20 08:58:10.155 ServerApp] Notebook pynsitu/dev/clean.ipynb is not trusted
[I 2022-10-20 08:58:19.983 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 08:59:46.613 ServerApp] Saving file at /pynsitu/pynsitu/geo.py
[I 2022-10-20 09:00:22.531 ServerApp] Saving file at /pynsitu/pynsitu/geo.py
[I 2022-10-20 09:00:30.959 ServerApp] Starting buffering for 4973cfa3-ee08-4aaa-86fd-cd75f65f51fc:8b9d8181-afa2-4914-8791-14478c4b31ce
[I 2022-10-20 09:00:31.703 ServerApp] Starting buffering for 2b65cd5b-80b7-472e-89d6-b084ccc92d78:1c105d7d-5bf0-4841-8047-36ac79dd2142
/home1/datahome/mdemol/.miniconda3/envs/m2env/lib/python3.8/contextlib.py:120: UserWarning: Creating scratch directories is taking a surprisingly long time. This is often due to running workers on a network file system. Consider specifying a local-directory to point workers to write scratch data to a local disk.
  next(self.gen)
/home1/datahome/mdemol/.miniconda3/envs/m2env/lib/python3.8/contextlib.py:120: UserWarning: Creating scratch directories is taking a surprisingly long time. This is often due to running workers on a network file system. Consider specifying a local-directory to point workers to write scratch data to a local disk.
  next(self.gen)
/home1/datahome/mdemol/.miniconda3/envs/m2env/lib/python3.8/contextlib.py:120: UserWarning: Creating scratch directories is taking a surprisingly long time. This is often due to running workers on a network file system. Consider specifying a local-directory to point workers to write scratch data to a local disk.
  next(self.gen)
/home1/datahome/mdemol/.miniconda3/envs/m2env/lib/python3.8/contextlib.py:120: UserWarning: Creating scratch directories is taking a surprisingly long time. This is often due to running workers on a network file system. Consider specifying a local-directory to point workers to write scratch data to a local disk.
  next(self.gen)
/home1/datahome/mdemol/.miniconda3/envs/m2env/lib/python3.8/contextlib.py:120: UserWarning: Creating scratch directories is taking a surprisingly long time. This is often due to running workers on a network file system. Consider specifying a local-directory to point workers to write scratch data to a local disk.
  next(self.gen)
[I 2022-10-20 09:02:00.169 ServerApp] Saving file at /pynsitu/pynsitu/geo.py
[I 2022-10-20 09:02:06.989 ServerApp] Kernel restarted: ec78af4b-f81e-4bc8-83c2-0fa64cefe642
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: e0311e8f-5fe0-4a1b-a76a-8e568ed9278f
[I 2022-10-20 09:02:20.095 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 09:02:59.950 ServerApp] Saving file at /pynsitu/pynsitu/geo.py
[I 2022-10-20 09:03:08.852 ServerApp] Kernel restarted: ec78af4b-f81e-4bc8-83c2-0fa64cefe642
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 534cc15b-d064-4a0d-8a9c-2fcaa586fbf4
[I 2022-10-20 09:03:50.694 ServerApp] Saving file at /pynsitu/pynsitu/geo.py
[I 2022-10-20 09:03:55.452 ServerApp] Kernel restarted: ec78af4b-f81e-4bc8-83c2-0fa64cefe642
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: d8b34825-1172-422d-ac06-6e0c8f4ff87f
[I 2022-10-20 09:04:20.216 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 09:06:20.336 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 09:06:46.623 ServerApp] Starting buffering for 0b9d85d6-8c2f-4aff-a016-569d3a83a0c4:f43a05e2-de4e-446d-a821-7b162822d67d
[I 2022-10-20 09:07:50.715 ServerApp] Saving file at /pynsitu/pynsitu/geo.py
[I 2022-10-20 09:08:20.527 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 09:15:50.733 ServerApp] Saving file at /pynsitu/pynsitu/geo.py
[I 2022-10-20 09:16:20.643 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 09:16:28.657 ServerApp] Kernel restarted: ec78af4b-f81e-4bc8-83c2-0fa64cefe642
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: e26a3455-7db9-4246-b0de-ca5da5f280ee
[I 2022-10-20 09:18:20.847 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[W 2022-10-20 09:21:28.684 ServerApp] WebSocket ping timeout after 119999 ms.
[W 2022-10-20 09:21:37.464 ServerApp] WebSocket ping timeout after 119998 ms.
[W 2022-10-20 09:21:37.464 ServerApp] WebSocket ping timeout after 119998 ms.
[I 2022-10-20 09:21:42.466 ServerApp] Starting buffering for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[I 2022-10-20 09:21:42.466 ServerApp] Starting buffering for ec78af4b-f81e-4bc8-83c2-0fa64cefe642:fcd7354c-0487-48c9-86a8-254ea1658a92
[I 2022-10-20 09:23:27.949 ServerApp] Restoring connection for 8566a462-1504-4dd8-8059-70ca8b08e2ed:10187e82-64d5-4a18-9904-ad4df75a791a
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 50bf8749-2f0d-4cd2-9efc-bd84f29584dc
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: ec98f933-1ae1-4cd9-bc21-95e25791c0b8
[I 2022-10-20 09:27:13.114 ServerApp] Saving file at /drifters_stats/sandbox/derivation.ipynb
[I 2022-10-20 09:28:28.710 ServerApp] Kernel restarted: ec78af4b-f81e-4bc8-83c2-0fa64cefe642
readline: /etc/inputrc: line 19: term: unknown variable name
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 5425d243-c2c7-4c58-9f8d-0a9f09c66519
[C 2022-10-20 09:28:46.919 ServerApp] received signal 15, stopping
Terminated
[I 2022-10-20 09:28:46.930 ServerApp] Shutting down 3 extensions
[I 2022-10-20 09:28:46.930 ServerApp] Shutting down 6 kernels
[I 2022-10-20 09:28:46.930 ServerApp] Kernel shutdown: 0b9d85d6-8c2f-4aff-a016-569d3a83a0c4
[I 2022-10-20 09:28:46.931 ServerApp] Kernel shutdown: 2b65cd5b-80b7-472e-89d6-b084ccc92d78
[I 2022-10-20 09:28:46.931 ServerApp] Kernel shutdown: c7732078-f7da-4834-a7b9-3048b41e0b41
[I 2022-10-20 09:28:46.931 ServerApp] Kernel shutdown: ec78af4b-f81e-4bc8-83c2-0fa64cefe642
[I 2022-10-20 09:28:46.932 ServerApp] Kernel shutdown: 4973cfa3-ee08-4aaa-86fd-cd75f65f51fc
[I 2022-10-20 09:28:46.932 ServerApp] Kernel shutdown: 8566a462-1504-4dd8-8059-70ca8b08e2ed
[I 2022-10-20 09:28:49.647 ServerApp] Shutting down 0 terminals
